.github/ISSUE_TEMPLATE/repo-create.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Repository creation/migration
+about: Create or migrate a repository into a Kubernetes Org
+
+---
+
+### New Repo, or migrate existing
+e.g. new repository
+
+### Requested name for new repository
+e.g. cloud-provider-foo
+
+### Which Organization should it reside
+e.g. kubernetes-sigs
+
+### Who should be provided access to this repository
+e.g (at)example_user should have write access
+
+### Has sig-architecture reviewed/approved this request?
+e.g. Yes, and here's the link.
```

Comment:
For kubernetes-sigs, sig-arch doesn't have a say, it's just the owning sig.

I am preferential to linkable proof in the form of a mailing list thread or meeting minutes.  I'm not going to accept a slack link.  Really don't know how I feel about a youtube link either.